index.js

```diff
@@ -359,6 +363,80 @@
       [a, b, a],
       function(x, y) { return x; });
 
+  //# A :: (a -> b) -> a -> b
+  //.
+  //. The A combinator. Takes a function and a value, and returns the result of
+  //. applying the function to the value. Equivalent to Haskell's `($)` function.
+  //.
+  //. ```javascript
+  //. > S.A(R.inc, 1)
+  //. 2
+  //.
+  //. > R.map(S.A(R.__, 100), [R.inc, Math.sqrt])
+  //. [101, 10]
+  //. ```
+  S.A =
+  def('A',
+      {},
+      [$.Function, a, b],
+      function(f, x) { return f(x); });
+
+  //# C :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. The C combinator. Takes a curried binary function and two values, and
+  //. returns the result of applying the function to the values in reverse.
+  //. Equivalent to Haskell's `flip` function.
+  //.
+  //. This function is very similar to [`flip`](#flip), except that its first
+  //. argument must be curried. This allows it to work with manually curried
+  //. functions.
+  //.
+  //. ```javascript
+  //. > S.C(R.concat, 'foo', 'bar')
+  //. 'barfoo'
+  //.
+  //. > R.filter(S.C(R.gt, 0), [-1, -2, 3, -4, 4, 2])
+  //. [3, 4, 2]
+  //. ```
+  S.C =
+  def('C',
+      {},
+      [$.Function, b, a, c],
+      function(f, x, y) { return f(y)(x); });
+
+  //# B :: (b -> c) -> (a -> b) -> a -> c
+  //.
+  //. The B combinator. Takes two functions and a value, and returns the result
+  //. of applying the first function to the result of applying the second to the
+  //. value. Equivalent to [`compose`](#compose) and Haskell's `(.)` function.
+  //.
+  //. ```javascript
+  //. > S.B(Math.sqrt, S.inc, 99)
+  //. 10
+  //. ```
+  S.B =
+  def('B',
+      {},
+      [$.Function, $.Function, a, c],
+      compose3);
+
+  //# S :: (a -> b -> c) -> (a -> b) -> a -> c
+  //.
+  //. The S combinator. Takes a curried binary function, a unary function, and a
+  //. value, and returns the result of applying the binary function to the value
+  //. along with the result of applying the unary function to the value. This
+  //. is known as the substitution combinator.
```

Comment:
This is the best human sentence I could come up with, unfortunately. I'm no native speaker. The following statement, however..:
`S(f, g, x)` is like `f(x, g(x))`
..Makes it very clear to me what's going on. We might consider adding that to the description to improve clarity, though I'm not sure if that would work for all people.